activate-virtualenv.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# ==========================================================================
+# Script for activating the virtualenv (creating it first, if necessary).
+# NOTE: Run this script while in the project root directory.
+#       It will not run correctly when run from another directory.
+# ==========================================================================
+
+# Set script to exit on any errors.
+set -e
+
+# Presumes the varialble VIRTUAL_ENV is already set to the name of the venv
+
+echo 'Activating virtualenv, if not already activated...'
+
+source $(brew --prefix)/bin/virtualenvwrapper.sh
```

Comment:
I _think_ my intent with this line was to handle the case where folks have it installed but haven't added it to their `.bash_profile`, but I can't remember for sure. In any case, I'm moving in the direction of your last suggestion, anyway, to address the other problem of the poor error handling with `source` invocation, so this will go away.